src/icons/service-serverless-azure.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 23 32" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m1.87 32 8.225-7.405v-1.871l-6.98 6.308v-2.908l15.013-13.649v2.93l-1.495 1.332 1.184.747 1.597-1.425v-5.663l-3.497.486 3.135-5.274V.632L18.357 0 0 15.895v3.89l5.89-.999-4.022 6.836V32h.001zM17.767 3.172V5.22l-3.365 5.87-1.713.239 5.078-8.158zM1.234 18.23v-2.101l7.167-.982-1.505 2.18-5.662.903z" fill="#FD5750"/><g clip-path="url(#a)"><path d="m17.103 18.084-3.833 3.323-3.27 5.87h2.949l4.154-9.193zm.51.778-1.636 4.61 3.136 3.94-6.084 1.045H23l-5.387-9.595z" fill="#0072C6"/></g><defs><clipPath id="a"><path fill="#fff" transform="translate(10 18.084)" d="M0 0h13v10.374H0z"/></clipPath></defs></svg>
```

Comment:
These are general change we make with svgs.
remove fill="none" in svg tag
change fill="currentColor" in path tag
Please make the change unless otherwise required

Reply:
There are multiple colors in this one, so can't do it.